src/loaders/ObjectLoader.js

```diff
@@ -466,8 +466,21 @@ THREE.ObjectLoader.prototype = {
 
 				case 'PerspectiveCamera':
 
-					object = Object.assign(
-							new THREE.PerspectiveCamera(), data );
+					object = new THREE.PerspectiveCamera(
+							data.fov, data.aspect, data.near, data.far );
+
+					if ( data.focus !== undefined ) object.focus = data.focus;
+					if ( data.zoom !== undefined ) object.zoom = data.zoom;
+
+					if ( data.filmGauge !== undefined ) {
+
+						if ( data.view !== null )
+							object.view = Object.assign( {}, data.view );
```

Comment:
> I think above THREE.Utils.assignOptionals (the name is probably bad), could safe the day when broadly applied in the loaders.

This is a good idea, especially if it can reduce the code size of ThreeJS.min.js :)  I'm all for it.

> extend without making assumptions about the completeness of exporters, and

I've found that this strategy in the past works well, to a point.  It is the standard assimulation vs. accomidation thing from psychology -- accomindate for as long as you can into the existing data structures until you need to do an assimulation revision: 

http://www.learningandteaching.info/learning/assimacc.htm